app/forms/new_phone_form.rb

```diff
@@ -88,6 +89,15 @@ def validate_not_voip
     end
   end
 
+  def validate_not_duplicate
+      current_user_phones = user.phone_configurations.map do |phone_configuration|
+        PhoneFormatter.format(phone_configuration.phone)
+      end
+
+      return unless current_user_phones.include?(phone)
+      errors.add(:phone, I18n.t('errors.messages.phone_duplicate'))
```

Comment:
this looks like it indents by 4, not our standard 2, to fix:

```suggestion
    current_user_phones = user.phone_configurations.map do |phone_configuration|
      PhoneFormatter.format(phone_configuration.phone)
    end

    return unless current_user_phones.include?(phone)
    errors.add(:phone, I18n.t('errors.messages.phone_duplicate'))
```